BeagleBone_black/i2c/i2c.hpp

```diff
@@ -0,0 +1,109 @@
+/*
+ * i2c.hpp
+ *
+ * Created: 10/02/2018 10:18:58
+ * Authors: M. Kristien, E. van Woerkom
+ */ 
+
+
+#ifndef I2C_HPP_
+#define I2C_HPP_
```

Comment:
The guard should be `BEAGLEBONE_BLACK_I2C_I2C_HPP_` according to the style guide

Reply:
I would suggest changing the style guide. We will not mingle files among different projects, so I would suggest using I2C_I2C_HPP_.
And how about using #pragma once?

Reply:
#pragma would also save us a lot of rewriting if we decide to move some files about.

Reply:
Did a little research on `#pragma once` and can't say I like it. It's non-standard and may have problems with links. It is not necessarily supported by all compilers although that should not be a problem anymore these days.

[Wikipedia](https://en.wikipedia.org/wiki/Pragma_once)